content/lessons/chapter-3/pool-3.tsx

```diff
@@ -179,104 +179,106 @@ export default function Pool3({ lang }) {
-                <Exponent
-                  className="font-space-mono font-normal"
-                  step={step}
-                  hashes={profile.hashes}
+    <div className="px-[20px] py-[30px] md:py-[75px]">
+      <div className="mx-auto flex max-w-[1340px] flex-col items-center gap-5 2xl:gap-8">
+        <HashrateChallenge
+          totalBlocks={TOTAL_BLOCKS}
+          blockRatio={BLOCK_RATIO}
+          step={step}
+          onStepUpdate={handleStepUpdate}
+          onProtagonistUpdate={handleProtagonsitBlockUpdate}
+          onAntagonistUpdate={handleAntagonsitBlockUpdate}
+          protagonists={PROTAGONISTS}
+          antagonists={ANTAGONISTS}
+          speed={5}
+          profiles={PROFILES.map((profile, i) => (
+            <Profile
+              key={i}
+              username={profile.username}
+              avatar={profile.avatar}
+              description={profile.description}
+            >
+              <Card className="flex">
+                <span
+                  className={clsx('fade-in font-nunito text-[15px] font-bold', {
+                    'text-white text-opacity-25': step === 0,
+                    'fade-in text-[#EDA081]': step !== 0,
+                  })}
+                >
+                  Blocks found
+                </span>
+                <span
+                  className={clsx(
+                    'fade-in font-space-mono text-[15px] font-normal text-white',
+                    {
+                      'text-opacity-25': step === 0,
+                    }
+                  )}
+                >
+                  {profile.value}
+                </span>
+              </Card>
+              <Card className="flex gap-4">
+                <span className="fade-in w-[103px] font-nunito text-[15px] font-bold text-[#EDA081]">
+                  Hashrate
+                </span>
+                <HashFrequency
+                  className="font-space-mono text-[15px]"
+                  disabled={false}
+                  step={0}
+                  hashPower={profile.hashpower}
                 />
+              </Card>
+              <Card className="flex">
+                <span
+                  className={clsx('fade-in font-nunito text-[15px] font-bold', {
+                    'text-white text-opacity-25': step === 0,
+                    'fade-in text-[#EDA081]': step !== 0,
+                  })}
+                >
+                  Hashes
+                </span>
+                <span className="fade-in font-nunito text-[15px] font-bold text-white text-opacity-25">
+                  <Exponent
+                    className="font-space-mono font-normal"
+                    step={step}
+                    hashes={profile.hashes}
+                  />
+                </span>
+              </Card>
+            </Profile>
+          ))}
+        >
+          <div className={`flex flex-col ${showText ? 'fade-in' : 'fade-out'}`}>
```

Comment:
adding the right padding instead of at the parent container would allow all other elements to have padding, not including the profilesContainer

Reply:
Can you please explain this differently, I don't full understand.

Reply:
This is related to the above comment relating to the profileContainer extending all the way to the edge of the screen, since we are rolling back that change this can be disregarded.

For posterity adding padding to this text container would mean all elements on the page EXCEPT the profileContainer would show padding